### ISSUE TITLE
We are missing CONTRIBUTING.md in this repository.

### ISSUE THREAD

**Issue by lotyp** [buggregator/trap#135]
We should take it from here:

* https://github.com/cycle/active-record/blob/master/.github/CONTRIBUTING.md

* https://github.com/wayofdev/laravel-package-tpl/blob/master/.github/CONTRIBUTING.md

**Comment by roxblnfk** [comment]
Also there is no CoC 🤔

**Comment by lotyp** [comment]
@roxblnfk also there are no ISSUE_TEMPLATES

**Comment by roxblnfk** [comment]
> @roxblnfk also there are no ISSUE_TEMPLATES

I think it's not required yet

**Comment by roxblnfk** [comment]
> @roxblnfk also there are no ISSUE_TEMPLATES

yeah, two simple templates (for feature requests and bug reports)
